### PR TITLE
Passing null for mapStateToProps

### PR DESCRIPTION
**Pull request by nattatorn-dev**
Thanks for sharing. be able to pass undefined or null as mapStateToProps (unsubscribe from redux store)

### CODE REVIEW

**Comment by Amin52J** [comment]
Thanks for contributing. Though I think it'd be more elegant to create another HOC (called something like reduxStatelessConnect or something like that) for passing null as mapStateToProps. 'cause the current implementation is gonna be needed sometimes.
Or I think even a better idea would be to make mapStateToProps an optional third argument to reduxConnect, if it wouldn't be `undefined` we'll use the third argument as mapStateToProps... Yeah let's just do that!
Will you please make the changes and commit again?

File: app/hoc/reduxConnect/index.js [hunk 1]
```diff
@@ -2,10 +2,6 @@
 
 import {connect} from 'react-redux';
 
-function mapStateToProps(state) {
-  return state;
-}
-
 export default function (WrappedComponent, actions) {
```
Comment (Amin52J) [inline app/hoc/reduxConnect/index.js]: This is where the third argument goes...

**Comment by nattatorn-dev** [comment]
Sure, I already updated. If the third argument is null, undefined or blank. I set mapStateToProps func as the default value.

**Comment by Amin52J** [comment]
Merged! Thanks again.

**Pull request merged**